File: src/info/daviot/dictionary/view/WordsListDisplay.java
package info.daviot.dictionary.view;

import java.awt.Font;

import javax.swing.event.ListSelectionListener;

public interface WordsListDisplay {

	void setSelectedIndex(int i);

	Object getSelectedValue();

	void setFont(Font font);

	void setSelectedValue(Object selectedValue, boolean shouldScroll);

	void addListSelectionListener(ListSelectionListener listSelectionListener);

}

File: src/info/daviot/dictionary/view/WordsListDisplay.scala
package info.daviot.dictionary.view;

import java.awt.Font;

import javax.swing.event.ListSelectionListener;

trait WordsListDisplay {

  def setSelectedIndex(i: Int);
  def getSelectedValue(): Any
  def setFont(font: Font);
  def setSelectedValue(selectedValue: Any, shouldScroll: Boolean);
  def addListSelectionListener(listSelectionListener: ListSelectionListener);

}
